src/modules/gameDevs/gameDevWebhookHandler.ts

```diff
@@ -16,12 +20,25 @@ import {
 export const getAchievementsThruWebhookHandler: Lifecycle.Method = async (request, h) => {
   const { gameType } = request.pre.webhookValidation;
   const achievements = await findAllAchievementsByGameType(gameType.id);
-  // if () {
-  //   throw Boom.forbidden('User is not the owner of the game');
-  // }
-  return h
-    .response({ achievements: achievements.map((achievement) => achievement.toJSON()) })
-    .code(200);
```

Comment:
@ccmoralesj note that I removed the "achievements: " format field since a client can already assume that he will receive the achievements resources by calling GET /achievements . wdyt?